### ISSUE TITLE
Does not prevent editing files

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/files_lock#25]
1. create a markdown or office file
2. share the file with user X
3. lock the file
4. log in as user X and edit the file with Text or ONLYOFFICE - no problems... Which is a problem ;-)

Updated-from-16 local test instance of Nc 18.0.1, 0.8.1 version of the app.

**Comment by daita** [comment]
Ok, this is a known issue that is related to Text and onlyoffice and their way to sharing collaborated modification on a file throw a temporary file/buffer:

- userA lock the file and open the file in Text App,
- userB open the file in Text App,
- userB can edit the file, but cannot save the modification on the filesystem, however the modification are sync on a temporary file and synchronize with the opened session from userA,
- userA's session autosave modification from both users on the filesystem.

tl;dr: userB's edit are shared to userA that have writing access to the file.

The solution is to have those apps to check if the file is manually/temporary locked: https://github.com/nextcloud/server/pull/18080

**Comment by daita** [comment]
(i am closing the issue as this cannot be done from the files_lock app, but have to be implemented in any collaborating app)
